pkg/sidecar_mounter/sidecar_mounter.go

```diff
@@ -104,6 +111,7 @@ func (m *Mounter) Mount(ctx context.Context, mc *MountConfig) error {
 	// so it is safe to force kill the gcsfuse process.
 	go func(cmd *exec.Cmd) {
 		<-ctx.Done()
+		klog.V(4).Infof("context marked as done, force killing gcsfuse process %d", cmd.Process.Pid)
```

Comment:
klog.V(4).Infof("context marked as done, sleep for 5 seconds, to evaluate gcsfuse process %d exit state)